site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-list.vm

```diff
@@ -523,10 +527,13 @@ function printPreview(target){
 						<td>$validator.escapeHtml($group.name)</td>
 						<td>
 							#if ($allowUpdateSiteMembership)
-							<form name="removeGroupForm" class="inlineForm" method="post" action="#toolForm("SiteAction")">
+							<form id="removeGroupForm" name="removeGroupForm" class="inlineForm" method="post" action="#toolForm("SiteAction")">
 								<input type="hidden" name="groupId" value="$group.id"/>
 								<input type="hidden" name="sakai_csrf_token" value="$sakai_csrf_token" />
 								<input type="submit" name="eventSubmit_doRemoveGroup" value="$tlang.getString("sitegen.siteinfolist.remove")"/>
+								<span id="spinnerRemove" style="display:none;">
+									<img class="loader" src="/../../library/newspicker/css/images/ajax-loader.gif" alt="Loading"/>
```

Comment:
Much better to just have `/library/...` as it the URL of the page includes more parts the link will break.